crates/component/src/lib.rs

```diff
@@ -5,18 +5,18 @@ mod bindings {
             "wasi:blobstore/blobstore@0.2.0-draft": generate,
             "wasi:blobstore/container@0.2.0-draft": generate,
             "wasi:blobstore/types@0.2.0-draft": generate,
-            "wasi:clocks/monotonic-clock@0.2.1": ::wasi::clocks::monotonic_clock,
-            "wasi:config/runtime@0.2.0-draft": generate,
-            "wasi:http/outgoing-handler@0.2.1": ::wasi::http::outgoing_handler,
-            "wasi:http/types@0.2.1": ::wasi::http::types,
-            "wasi:io/error@0.2.1": ::wasi::io::error,
-            "wasi:io/poll@0.2.1": ::wasi::io::poll,
-            "wasi:io/streams@0.2.1": ::wasi::io::streams,
+            "wasi:clocks/monotonic-clock@0.2.2": ::wasi::clocks::monotonic_clock,
+            "wasi:config/store@0.2.0-draft": generate,
```

Comment:
We have cases of the older named interface being used. Can we support both for an interim period (like one month)?

Reply:
We do support both, actually

Reply:
I confused this block with a different location in code, no changes necessary, everything is in order here.
`wasi:config/runtime` bindings are generated separately if that's the source of confusion here. It's impossible to generate *both* `wasi:config/store` and `wasi:config/runtime` without patching in a single invocation, since there's a dependency clash in that case (2 versions of the same package)